processor/src/main/java/com/linecorp/decaton/processor/runtime/LoggingContext.java

```diff
@@ -19,7 +19,7 @@
 import org.slf4j.MDC;
```

Comment:
I think this class should be placed in root because we use it when implementing DecatonProcessor to output logs with MDC infos.

Reply:
This too, nice catch.